src/lib.rs

```diff
@@ -239,6 +239,13 @@ impl Counter for u64 {
     fn as_u64(&self) -> u64 { *self }
 }
 
+impl Counter for usize {
+    #[inline]
+    fn as_f64(&self) -> f64 { *self as f64 }
+    #[inline]
+    fn as_u64(&self) -> u64 { *self as u64 }
+}
```

Comment:
This is probably safe to add, though I haven't thought through it carefully yet. However, I'm not clear why this is desirable. Can you maybe explain your use-case a little more? Typically people would choose a counter type based on the sample rate and the duration that a single histogram would be accumulating data. Why is it desirable to have the maximum count per bucket change when the architecture changes?

Reply:
Initially I added `usize` because the `tacho` library uses a `usize` counter, and I'm updating it from hdrsample 3.x to 4.x.  That being said, I think your question is valid, and I didn't quite think this through.  It's likely that `tacho` should be using a fixed size value.  I'm going to strip this commit from the PR.

Reply:
Ah, makes sense. Glad to see this data structure being used for monitoring stuff!